### PR TITLE
Fix missing folder for links in the database

### PR DESCRIPTION
**Pull request by arjaz**
Now we will create it.
Also, we should probably move that to the database module somehow

### CODE REVIEW

File: src/Handlers.hs [hunk 1]
```diff
@@ -53,8 +53,10 @@ startApp = do
   appDbPath <- getEnv "appDbPath"
   let userPath = appDbPath <> "/user"
       aliasPath = appDbPath <> "/alias"
```
Comment (alnyan) [inline src/Handlers.hs:55]: There's no need for this line. xxx`Fs`  implementations use "link" directory for storing aliases, not "alias".

**Review by alnyan — COMMENTED** [review]
General comment: I suggest we really move the "prepare database" functionality to db typeclass.

File: src/Handlers.hs (hunk 1 above)
Comment (arjaz) [reply src/Handlers.hs]: Ok, I fixed that in the following commit

**Review by alnyan — APPROVED** [review]
Good to go, waiting for CI to finish

**Pull request merged**